icons/tent.svg

```diff
@@ -9,9 +9,8 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M19 20 10 4" />
-  <path d="m5 20 9-16" />
+  <path d="M4.5 20 14 4" />
+  <path d="M19.5 20 10 4" />
   <path d="M3 20h18" />
-  <path d="m12 15-3 5" />
-  <path d="m12 15 3 5" />
+  <path d="m15 20-3-5-3 5" />
```

Comment:
I think we could easily bump this up in size to better match our ideal visual weight:
![image](https://github.com/lucide-icons/lucide/assets/17746067/73777892-246e-4079-84a4-7ef63c595864)
```svg
  <path d="M3.5 21 14 3" />
  <path d="M20.5 21 10 3" />
  <path d="M2 21h20" />
  <path d="M15.5 21 12 15l-3.5 6" />
```

What do you think?